docs/topics/Gateway.md

```diff
@@ -563,10 +563,30 @@ A user's presence is their current state on a guild. This event is sent when a u
 |-------|------|-------------|
 | user | [user](#DOCS_USER/user-object) object | the user presence is being updated for |
 | roles | array of snowflakes | roles this user is in |
-| game | object | null, or an object containing one key of "name" |
+| game | ?[game](#DOCS_GATEWAY/game-object) object | the game being played/streamed, or null |
 | guild_id | snowflake | id of the guild |
 | status | string | either "idle", "online" or "offline" |
 
+### Game Object
+
+###### Game Structure
+
+| Field | Type | Description | Present |
+|-------|------|-------------|---------|
+| name | string | the game's name | Always |
+| type | ?integer | 0 is default, 1 is a twitch stream, 2 is a youtube stream  | Sometimes |
```

Comment:
This should be broken out into a subtable as well "Game Types", also they key is as such:

| ID | Name | Desc |
|----|----------|---------|
| 0 | Game | "Playing {}" |
| 1 | Twitch | "Streaming {}" |

feel free to format in another way.